src/main/java/frc/robot/RobotContainer.java

```diff
@@ -266,9 +259,16 @@ private void configureButtonBindings() {
         .pov(270)
         .whileTrue(climber.raiseOrLower(ClimberConstants.CLIMBER_BACKWARD_PERCENT));
 
-    driverController.rightBumper().whileTrue(arm.toSetpoint(ArmConstants.STOW));
+    // driver controller
+    driverController.rightBumper().onTrue(arm.toSetpoint(ArmConstants.STOW));
     driverController.leftBumper().onTrue(orchestrator.alignArmAmp());
-    //    driverController.leftBumper().whileTrue(orchestrator.alignArmSpeaker());
+    driverController.leftTrigger().toggleOnTrue(orchestrator.intakeBasic());
+    driverController.rightTrigger().onTrue(orchestrator.indexBasic());
+    driverController.a().onTrue(Commands.runOnce(() -> drive.stopWithX()));
+    driverController.b().onTrue(climber.setRatchet(false));
```

Comment:
Does this work when you also have operatorController.back().whileTrue and operatorController.back().whileFalse?

Reply:
yes, we used something similar last year:
`  // Set the game piece type
    operatorController.back().whileFalse(new WantConeCMD(intakeRelease));
    operatorController.back().whileTrue(new WantCubeCMD(intakeRelease));`

Reply:
wait, i see your point. this won't work

Reply:
I'll remove the driver commands